dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -1188,12 +1203,13 @@ def pipe(max_batch_size, input_data, device, input_layout=None):
     "readers.nemo_asr",         # readers do do not support variable batch size yet
     "readers.video",            # readers do do not support variable batch size yet
     "readers.video_resize",     # readers do do not support variable batch size yet
-
 ]
 
 def test_coverage():
-    methods = module_functions(fn, remove_prefix = "nvidia.dali.fn")
+    methods = module_functions(fn, remove_prefix="nvidia.dali.fn")
     methods += module_functions(dmath, remove_prefix = "nvidia.dali")
-    covered = tested_methods + excluded_methods
+    exclude = "|".join(["(^" + x.replace(".", "\.").replace("*", ".*").replace("?", ".") + "$)" for x in excluded_methods])
```

Comment:
Do we need this `$` or it is just in case?

Reply:
It's formally necessary, even if not needed now - otherwise if we put, say, "paste", here, it would match also "multi_paste" and "bbox_paste".

Reply:
Rather "paste_x" and "paste_y". `^` should make sure of that. Anyway, just asking.